tt_metal/impl/device/device.cpp

```diff
@@ -3691,18 +1582,14 @@ void Device::enable_async(bool enable) {
 }
 
 bool Device::using_slow_dispatch() const {
-    return !using_fast_dispatch();
-}
-
-bool Device::using_fast_dispatch() const {
-    return using_fast_dispatch_;
+    return not (this->using_fast_dispatch_);
 }
 
 void Device::begin_trace(const uint8_t cq_id, const uint32_t tid) {
     ZoneScoped;
     TracyTTMetalBeginTrace(this->id(), tid);
     TT_FATAL(!this->hw_command_queues_[cq_id]->tid.has_value(), "CQ {} is already being used for tracing tid {}", (uint32_t)cq_id, tid);
-    this->mark_allocations_safe();
+    this->MarkAllocationsSafe();
```

Comment:
why are we changing casing? Even for camel case, first `m` should be lower case

Reply:
Bad rebase I think, will fix

Reply:
Should be fixed now